src/design-system/assets-table/assets-table-token-amount.component.tsx

```diff
@@ -5,22 +5,26 @@ import { Text } from '../text';
 
 import * as cx from './assets-table-token-amount.css';
 
+import type { TypographyVariants } from '../text/text.css';
+
 interface Props {
   amount: string;
-  fiatPrice: string;
+  details: string;
+  detailsColor?: TypographyVariants['color'];
 }
 
 export const TokenAmount = ({
   amount,
-  fiatPrice,
+  details,
+  detailsColor = 'secondary',
 }: Readonly<Props>): JSX.Element => {
   return (
     <Flex flexDirection="column" alignItems="flex-end" className={cx.container}>
       <Text.Body.Large weight="$semibold" data-testid="token-amount">
         {amount}
       </Text.Body.Large>
-      <Text.Body.Normal color="secondary" weight="$semibold">
-        {fiatPrice}
+      <Text.Body.Normal color={detailsColor} weight="$regular">
+        {details}
```

Comment:
how about allowing to customize **weight** as well? maybe we can have styling defined as
```
detailsStyle: {
  color?: TypographyVariants['color'],
  weight?: ...
}
```

and weight would be default to '$regular'